src/styles/utils/resets-normalize.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
- Don't set margins/paddings to all elements, reset only for tags that actually need it
- The same regarding font-family, instead you can specify it to html or body tag

Reply:
> * Don't set margins/paddings to all elements, reset only for tags that actually need it
> * The same regarding font-family, instead you can specify it to html or body tag

I've moved font-family from `*` to the `html `tag, but I wanna do full reset of margins and padding because I knew that I'm gonna set it manually through all my work. So that's the reason - I don't wanna get errors with test because of some random padding of li or ul or some other elements.

Reply:
> > * Don't set margins/paddings to all elements, reset only for tags that actually need it
> > * The same regarding font-family, instead you can specify it to html or body tag
> 
> I've moved font-family from `*` to the `html `tag, but I wanna do full reset of margins and padding because I knew that I'm gonna set it manually through all my work. So that's the reason - I don't wanna get errors with test because of some random padding of li or ul or some other elements.

Ok, but it's better to reset only for tags that actually have them